.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+<!--
+- Please ensure code changes are split into a series of logically independent commits.
+- Every commit should have a subject/title (What) and a description/body (Why).
+- Every PR must have a description.
+- As an example you can use git commit -m"What" -m"Why" to achieve the requirements above. GitHub automatically recognises the commit description (-m"Why") in single commit PRs and adds it as the PR description.
+- If the PR includes auto generated code, make sure to add this code in different commit (last commit), to make the review process easier.
+-->
+
+**What this PR does / why we need it**:
+
+**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story*:
+Fixes #
+
+**Change type**
+- [ ] New feature
+- [ ] Bug fix
+
+**Impacted elements**
+- [ ] Unit tests
+- [ ] Integration tests
+- [ ] CI
+- [ ] Docs
+- [ ] Auto generated SDK code
+
+**Impacted services**
+- [ ] Workflow Service
+- [ ] Notifivcation Service
+
+**Checklist**
+- [ ] Subject and description added to both, commit and PR.
+- [ ] Relevant issues have been referenced.
+- [ ] `mvn clean install` was executed to validate changes
+- [ ] Integration tests were validated with the change
```

Comment:
These two will be run by the CI, I don't think there is a need to check them manually

Reply:
But it can be good reminder for the Author, before the PR is opened

Reply:
it is implied by `**Impacted elements**` section

Reply:
As I see it, `**Impacted elements**` indicates if the element code was changed (i.e. changed integration test)
But even if the integration code is not changed, it need to be validated

Any way, I'm flexible on this matter and it can be eliminated to simplify the template.

Reply:
Removed those lines